php-fpm/recipes/cloudwatch.rb

```diff
@@ -0,0 +1,22 @@
+cronscript = '/opt/easybib/bin/phpfpm-cloudwatch.sh'
+
+package 'awscli'
+
+template cronscript do
+  source 'cloudwatch.sh.erb'
+  mode   '0755'
+  owner  'root'
+  group  'root'
+  variables(
+    'instancename' => get_hostname.gsub(/\W/, '_'),
```

Comment:
@gilleyj if "dots" are not allowed, I would still use `get_record_name` and then replace on that.